dataeng/jobs/analytics/AnalyticsExporter.groovy

```diff
@@ -129,11 +133,12 @@ class AnalyticsExporter {
         }
 
         dslFactory.job('analytics-exporter-master') {
+            description('The Analytics Exporter weekly job, which exports tons of structure and state data for every course for every org and delivers them encrypted to our partners via S3.  Specifically, this sets up the shared edx-platform execution environment, fetches a list of all the orgs, then kicks off downstream analytics-exporter-worker jobs for each one.')
```

Comment:
Technically it only exports data for orgs who are at the partner level or above.